zcash_client_sqlite/src/testing.rs

```diff
@@ -234,6 +234,35 @@ where
         (res, nf)
     }
 
+    /// Creates a fake block at the expected next height including the wallet transaction
```

Comment:
This overlaps somewhat with @daira's changes in https://github.com/zcash/librustzcash/pull/908

Reply:
Gah, it's exactly duplicative. But I don't have access to those changes because they were merged into `main`.

Reply:
Yep, this directly conflicts with `main`. AFAICT the only PRs that `main` has which the release branch doesn't are #908, #972 (no-op), and #954, so I propose that we merge `main` into the release branch (highly unconventional for a release branch, but AFAICT safe) and then I rebase on it.

Reply:
Okay, agreed.